server/migrator/MigratorClient.java

```diff
@@ -106,4 +118,80 @@ public boolean success() {
             return success;
         }
     }
+
+    private static class ProgressPrinter {
```

Comment:
This class is only moved here, not modified. This is to make the top level classes less noisy.